cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlMain.java

```diff
@@ -47,6 +48,14 @@ public static void main(String[] args) throws Exception {
         printErrorMessageAndDie();
       }
     }
+    // Future ssl bits
+    String protocol = "http";
+
+    // For security reasons listen on the loopback address by default
+    String hostname = "127.0.0.1";
```

Comment:
Nit: For clarity, would it be possible to move the hardcoded value to a variable? -- i.e. 
```
private static final String DEFAULT_HOSTNAME = "127.0.0.1";
...
String hostname = DEFAULT_HOSTNAME;
```

Reply:
I was following the approach taken for `port` variable. But, lemme fix all the relevant ones as well.